plugins/hydra_ax/hydra_plugins/ax_sweeper/conf/hydra/sweeper/ax.yaml

```diff
@@ -0,0 +1,45 @@
+hydra:
+  # See https://ax.dev/
+  ax:
+    max_trials: 100
+    early_stop:
+      # Number of epochs without a significant improvement from
+      # the currently known best parameters
+      # An Epoch is defined as a batch of trials executed in parallel
+      max_epochs_without_improvement: 10
+      # An improvement larger than this is considered significant
+      epsilon: 0.00001
+
+      minimize: ${hydra.ax.experiment.minimize}
+
+    experiment:
+      # Experiment name
+      name: ${hydra.job.name}
+      # Name of metric to optimize or null if you only return one metric
+      objective_name: objective
+      # Minimize objective:
+      #   true: minimize
+      #   false: maximize
+      minimize: false
+      parameter_constraints: null
+      outcome_constraints: null
+      status_quo: null
+
+  sweeper:
+    class: hydra_plugins.ax_sweeper.AxSweeper
+    params:
+      experiment: ${hydra.ax.experiment}
+      early_stop: ${hydra.ax.early_stop}
+      verbose_logging: false
+      max_trials: ${hydra.ax.max_trials}
+      # set random seed here to make Ax results reproducible
+      random_seed: null
+      ax_params:
+        banana.x:
+          type: range
+          bounds: [-5.0, 10.0]
+          log_scale: false
+        banana.y:
+          type: range
+          bounds: [-5.0, 5.0]
+          log_scale: false
```

Comment:
This should be in the banana example config file, not the core plugin config file.
Also, remember hydra.ax block?
this can point to it.

banana.yaml:
```yaml
hydra:
  ax:
    params:
      banana.x:
        type: range
        bounds: [-5.0, 10.0]
        log_scale: false
      banana.y:
        type: range
        bounds: [-5.0, 5.0]
        log_scale: false 
```

ax.yaml
```yaml
hydra:
  sweeper:
    ...
    params:
      params: ${hydra.ax.params}
```


Please document that those are optional (and what is the default behavior if they are missing).

Reply:
Is it okay to put those comments in the config file itself or should they be in README?

Reply:
Probably both.
I would take out the actual values here, but the README should document each of the fields (type, bounds and log_scale).

What if the type is fixed? it's still called bounds?

Reply:
or choice

Reply:
Each type has a different set of keys (eg 'range' -> 'bounds', 'fixed' -> 'value' etc). Will document them in the README.